### ISSUE TITLE
Photos save.

### ISSUE THREAD

**Issue by kylerchin** [justgoaway/justgoaway.github.io#1]
![20160814_175405](https://cloud.githubusercontent.com/assets/7539174/18456913/a2baba78-7909-11e6-96ea-45b6a567c3cb.jpg)
![20160818_123401](https://cloud.githubusercontent.com/assets/7539174/18456915/a2cb6562-7909-11e6-8c5c-d81b6e98c802.jpg)
![20160818_123429](https://cloud.githubusercontent.com/assets/7539174/18456918/a2ed53c0-7909-11e6-81d2-78481ac03588.jpg)
![20160818_183727](https://cloud.githubusercontent.com/assets/7539174/18456916/a2e357c6-7909-11e6-997a-86c911d95ffb.jpg)
![20160818_194542](https://cloud.githubusercontent.com/assets/7539174/18456920/a2fac29e-7909-11e6-9893-c4d70215a8a3.jpg)
![20160818_194546](https://cloud.githubusercontent.com/assets/7539174/18456919/a2f75ad2-7909-11e6-99ed-fc5e3dfeaa16.jpg)
![20160818_161028](https://cloud.githubusercontent.com/assets/7539174/18456922/a56267d0-7909-11e6-9d0b-a17ad0118430.jpg)
![2016-09-18_14 41 20-volta4](https://cloud.githubusercontent.com/assets/7539174/18900658/9977674a-84f8-11e6-97cb-89f2e965a6f4.png)
![2016-09-18_14 41 20](https://cloud.githubusercontent.com/assets/7539174/18900659/9a4e6c54-84f8-11e6-93ec-84b803c2bdd0.png)

**Comment by ghost** [comment]
those are some neat pictures

**Comment by kylerchin** [comment]
thanks. I just use github for static picture hosting. A trick I learned long time ago.

**Comment by ghost** [comment]
@kylerschin, I'd provide you with some sftp storage if you want

**Comment by kylerchin** [comment]
@R41z I mainly just use github for the fast speed. What is your speed? Also, do you have a CDN? If so, I would be glad to have some sftp storage!

**Comment by ghost** [comment]
No, I do not have CDN, I'm not into bloat, anyways, the server is hosted in Paris, if you are willing to test speed, use this: http://raiz.duckdns.org/100mb.bin

Raiz

**Comment by ghost** [comment]
Oh! and by the way, stop using this issue as a chatting platform, consider contacting me via: http://raiz.duckdns.org/contact.html

**Comment by kylerchin** [comment]
Cool.
